get-message-service-bus-queue-peek-lock-renew-sequential-convoy/manifest.json

```
@@ -1,9 +1,9 @@
 {
-    "title": "Azure Service Bus: Get message from a queue with token renewal - Peek lock renew pattern",
+    "title": "Azure Service Bus: Sequential convoy pattern",
 
-    "description": "This workflow template triggers when a message is available in a Service Bus queue. The workflow gets the message from the queue, locks the message for processing, and renews the lock token until processing completes. The message is deleted from the queue after processing completes. This pattern helps you make sure that the workflow processes the message only once and that the message doesn't get lost if a failure happens during processing.",
+    "description": "This workflow template triggers when a message is available in a Service Bus session enabled queue. The workflow processes related messages from a specific session on the same workflow instance, creating different workflow instances for each session in the queue.",
 
-    "prerequisites": "Before you can use this template, you need to create or set up a Service Bus namespace with a queue. For more information, see [Create a Service Bus namespace and a queue](https://go.microsoft.com/fwlink/?linkid=2296007).",
+    "prerequisites": "Before you can use this template, you need to configure a Service Bus namespace and create a queue with session enabled. For more information about Service Bus see [Create a new queue on Service Bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal). You also need to configure your Logic App to support stateful connectors, which is required for sessions. For more information see [Enable stateful mode for stateless built-in connectors in Azure Logic Apps](https://go.microsoft.com/fwlink/?linkid=2296010)",

```

@wsilveiranz: Was there a reason for not using the previously provided FW link? We shouldn't be using hardcoded doc URLs with the locale (en-us) included.

```suggestion
    "prerequisites": "Before you can use this template, you need to set up a Service Bus namespace and create a queue with sessions enabled. For more information, see [Create a Service Bus namespace and a queue](https://go.microsoft.com/fwlink/?linkid=2296007) and [Enable message sessions for a Service Bus queue](https://go.microsoft.com/fwlink/?linkid=2296202). You also need to set up your logic app to support stateful connectors, which is required for sessions. For more information, see [Enable stateful mode for stateless built-in connectors in Azure Logic Apps](https://go.microsoft.com/fwlink/?linkid=2296010).",

```

---

The first link was a copy and paste error.